src/Native/Runtime/GCMemoryHelpers.inl

```diff
@@ -7,14 +7,21 @@
 // Unmanaged GC memory helpers
 //
 
+#ifndef DACCESS_COMPILE
```

Comment:
Yes.  I started porting this file to DACCESS_COMPILE, but then stopped.  I didn't see any harm in being consistent with the use of GPTR_DECL as the declaration.  In fact, I kind of like being consistent, so I left these as GPTR_DECL.